src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -12,7 +12,6 @@ import * as nls from 'vs/nls';
 import * as panel from 'vs/workbench/browser/panel';
 import * as platform from 'vs/base/common/platform';
 import { Extensions, IConfigurationRegistry } from 'vs/platform/configuration/common/configurationRegistry';
-import { GlobalQuickOpenAction } from 'vs/workbench/browser/parts/quickopen/quickopen';
```

Comment:
It had a reference to the Action. As I removed the action, this had to be removed as well